socorro/scripts/fetch_crashids.py

```diff
@@ -177,17 +183,31 @@ def main(argv=None):
     params['_columns'] = 'uuid'
 
     # Override with date if specified
-    datestamp = args.date
-    if 'date' not in params or datestamp != 'yesterday':
-        if datestamp == 'yesterday':
+    if 'date' not in params or args.date:
+        datestamp = args.date or 'yesterday'
+
+        if datestamp == 'today' or datestamp == 'now':
+            startdate = utc_now()
+        elif datestamp == 'yesterday':
             startdate = utc_now() - datetime.timedelta(days=1)
         else:
             startdate = datetime.datetime.strptime(datestamp, '%Y-%m-%d')
 
         enddate = startdate + datetime.timedelta(days=1)
+
+        startdate = startdate.strftime('%Y-%m-%d')
+        enddate = enddate.strftime('%Y-%m-%d')
+
+        if datestamp == 'now':
+            # If we want crashes for now, add some cache-busting bits to the
+            # url and also sort in reverse by date
```

Comment:
Why sort in reverse by date?

Reply:
If you don't sort reverse by date, then you don't get the most recent crashes.